scripts/link-data.sh

```diff
@@ -76,6 +78,7 @@ link_locs=(
   scRNA-seq/data/tabula-muris/mm_ensdb95_tx2gene.tsv
   scRNA-seq/gene-sets
   scRNA-seq-advanced/data/PBMC-TotalSeqB/raw_feature_bc_matrix
+  scRNA-seq-advanced/data/SCPCP000005-samples
```

Comment:
This will link won't work, as the directory will already exist. What we will probably want is to link to a different subdirectory which contains the processed files.